CHANGELOG.md

```diff
@@ -20,6 +20,9 @@
 - Added Vibrato and Sidechain patch cables to Automation View Overview and Grid Shortcuts
 - Added ability to enter clips in `Song Grid View Green Mode` by `Pressing a Clip Pad` + `Pressing the Clip button` if you have `Select in Green Mode` enabled in the `SETTINGS > DEFAULTS > UI > SONG > GRID` menu.
 
+#### <ins>Instrument View</ins>
```

Comment:
Just a nit, call it Instrument Clips

Reply:
Ok, by the way. What do you think about changing from Learn to Load as the toggle button?
In my faceplate i have Learn labels as Learn/Copy so i related it to "copying the sample to the synth", but maybe Load is much more obvious for a user

Reply:
tried with LOAD for the toggle and feels nice and intuitive

Reply:
Ok changes addressed and bonus point: fixed issue with SCW not loading as that when osc1type is already wavetable